rocAL_pybind/rocal_pybind.cpp

```diff
@@ -724,5 +724,15 @@ PYBIND11_MODULE(rocal_pybind, m) {
           py::return_value_policy::reference);
     m.def("toDecibels", &rocalToDecibels,
           py::return_value_policy::reference);
+    m.def("resample", &rocalResample,
```

Comment:
@SundarRajan28 Where are the magic methods definition?

Please add __mul__ and __add__ here. 
Else we will get error when we are trying to multiply the uniform dist with the scalar float